homeassistant/components/overkiz/climate_entities/somfy_heating_temperature_interface.py

```diff
@@ -0,0 +1,179 @@
+"""Support for Somfy Heating Temperature Interface."""
+from __future__ import annotations
+
+from typing import Any
+
+from pyoverkiz.enums import OverkizCommand, OverkizCommandParam, OverkizState
+
+from homeassistant.components.climate import (
+    PRESET_AWAY,
+    PRESET_COMFORT,
+    PRESET_ECO,
+    PRESET_NONE,
+    ClimateEntity,
+    ClimateEntityFeature,
+    HVACAction,
+    HVACMode,
+)
+from homeassistant.const import ATTR_TEMPERATURE, UnitOfTemperature
+
+from ..coordinator import OverkizDataUpdateCoordinator
+from ..entity import OverkizEntity
+
+OVERKIZ_TO_PRESET_MODES: dict[str, str] = {
+    OverkizCommandParam.SECURED: PRESET_AWAY,
+    OverkizCommandParam.ECO: PRESET_ECO,
+    OverkizCommandParam.COMFORT: PRESET_COMFORT,
+    OverkizCommandParam.FREE: PRESET_NONE,
```

Comment:
```suggestion
    OverkizCommandParam.FREE: PRESET_NONE,
    OverkizCommandParam.FROSTPROTECTION: PRESET_FROST_PROTECTION,
```
It seems this device works with frost protection, as https://github.com/home-assistant/core/blob/dev/homeassistant/components/overkiz/climate_entities/atlantic_electrical_heater.py or https://github.com/home-assistant/core/blob/dev/homeassistant/components/overkiz/climate_entities/atlantic_pass_apc_heating_zone.py or https://github.com/home-assistant/core/blob/dev/homeassistant/components/overkiz/climate_entities/valve_heating_temperature_interface.py

Reply:
That can be done in a new PR as it needs tests.